pkg/customers/approval_ofac.go

```diff
@@ -108,6 +100,7 @@ func getLatestCustomerOFACSearch(logger log.Logger, repo CustomerRepository) htt
 			moovhttp.Problem(w, err)
 			return
 		}
+		result.Blocked = result.Match > ofacMatchThreshold
```

Comment:
This one is interesting, because for the same OFAC search this endpoint may return different results. `Blocked` depends on `ofacMatchThreshold`. If we change it then we may face the case when user is blocked but `getLatestCustomerOFACSearch` returns Blocked: false...

I think that we should store Blocked when it was calculated and then return it from DB not calculate it again.

Reply:
I was wondering about this. Thanks for bringing it up Pavel. I think calling the db column "blocked at search time" or something makes sense. 

Correct me, but @joshsadler would you expect the OFAC searches to be immutable? aka `blocked: <boolean>` represents the block status _at the time of searching_ or the current block status? 

In most cases these two scenarios won't be different, but we should clarify.

Reply:
sorry I missed this ping @adamdecaf. I'm thinking it should be block status at the time of searching. That seems the least restrictive.